string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-unused-vars */
+
+function titleCase(string) {
+  var wordsInTitle = string.split(' ');
+
+  var newTitle = [];
+
+  if (wordsInTitle[0].length !== 1) {
+    newTitle.push(wordsInTitle[0].charAt(0).toUpperCase() + wordsInTitle[0].slice(1, wordsInTitle[0].length));
+  } else {
+    newTitle.push(wordsInTitle[0].charAt(0).toUpperCase());
+  }
+
+  var singleWords;
+
+  for (var i = 1; i < wordsInTitle.length; i++) {
+    if (wordsInTitle[i].length > 3) {
+      newTitle.push(concatSingleWord(wordsInTitle[i].split('')));
+    } else if ((wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 'a') ||
+      (wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 'b') ||
+      (wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 'f') ||
+      (wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 'i') ||
+      (wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 'o') ||
+      (wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 'p') ||
+      (wordsInTitle[i].length < 4 && wordsInTitle[i].charAt(0).toLowerCase() === 't')) {
```

Comment:
This is very confusing. If you just make an array of all short words, you can check if the current word is one of those words. It is not guaranteed that all three-letter words are "short" words. Plus, your `titleCase` function should theoretically handle any input, so these super-specific checks shouldn't be part of your strategy.